### ISSUE TITLE
GitHub repo link dead?

### ISSUE THREAD

**Issue by dandv** [cdotyone/mochaui#152]
The link at the top of [this page](https://github.com/mui/mochaui) directs to some parked domain site.

**Comment by robernet** [comment]
You're correct. The website's owner hasn't update it. It's unfortunated since MochaUI is a great piece of work. The code's been updated to the latest Mootools version, some code's been fixed from really good collaborators. Hope you can give it a try.

**Comment by dandv** [comment]
Well, there are [so many JS widget libraries](http://stackoverflow.com/a/2144878/1269037) out there, that I'd like to see a demo first.

**Comment by gitowiec** [comment]
Is this project dead or just something will happen very soon? I am looking for demo page.

**Comment by robernet** [comment]
Hello Marek,

MochaUI has been an interesting project for me and I'm happy to contribute.
My latests goals were on making it work with the newest MooTools versions
and browsers. IE was a nightmare and I definitly quit on that one, and I
submitted some changes to the MooTools community in order to handle touch
events properly, since Mocha uses drag and drop.
My goal now is trying to adjust MochaUI to work with mobiles and tablets,
which hasn't been easy to be honest.

I can't give you a timetable, because I don't have one. Sorry about that.

More than a dead project, I'll like to define it as a walking dead project.
Still struggling.

Regards,
Roberto
